app/src/components/Task/MyTasks/index.tsx

```diff
@@ -61,7 +67,7 @@ function MyTasks({ projectId, title }: IMyTasksProps) {
         </Menu.Dropdown>
       </Menu>
       <div className='w-full px-5 py-3'>
-        <div className='flex items-center text-xl mb-3 font-monaspace'>
+        <div className='flex items-center text[0.875rem] mb-3 font-monaspace font-normal'>
```

Comment:
Should be `text-[0.875rem]`

But better use `text-sm` (it's the same 0.875rem)

Reply:
yes don it and add it